### ISSUE TITLE
Bitdefender identifies FanControl as threat

### ISSUE THREAD

**Issue by Capad** [Rem0o/FanControl.Releases#2297]
**Describe the bug**
The file C:\FanControl_net_7_0\FanControl.exe is infected with Trojan.GenericKD.71462215.

Version of FC 180, updated a day before.

**Is there a log.txt file next to FanControl.exe with recent date entries?**
Not from today, when problem occurred.

**Relevant hardware specs and setup**
Everything vanilla, custom build, compatibility OK.

**Comment by Rem0o** [comment]
False positive.

**Comment by andlindt** [comment]
Yeah, I also have Bitdefender and I get them alarm threats too from time to time, even with all the requiered exceptions set. That's life haha

**Comment by TempleKa** [comment]
Also started getting this from Bitdefender a couple of days ago after updating FanControl version.

**Comment by WyvernIXTL** [comment]
Yep, have the same issue.

**Comment by andlindt** [comment]
> Yep, have the same issue.
> 
> _I've reported it as false positive [here](https://www.bitdefender.com/consumer/support/answer/29358/). The more the merrier, right?_

Gonna do that to.

**Comment by WyvernIXTL** [comment]
Ok, it has been a few days and bitdefender did not report it on a quick scan.

I wonder if this issue will crop up on every update.

**Comment by johbec** [comment]
Same for 1.8.1 using VirusTotal.
